-------------------------------------------------------------------------------
Test set: app.ExamplesOfAssertionsTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.100 s <<< FAILURE! -- in app.ExamplesOfAssertionsTest
app.ExamplesOfAssertionsTest.simpleAssertionExamples -- Time elapsed: 0.056 s <<< FAILURE!
org.opentest4j.AssertionFailedError: Unexpected exception type thrown, expected: <java.lang.RuntimeException> but was: <java.lang.Error>
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:67)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:35)
	at org.junit.jupiter.api.Assertions.assertThrows(Assertions.java:3115)
	at app.ExamplesOfAssertionsTest.simpleAssertionExamples(ExamplesOfAssertionsTest.java:35)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
Caused by: java.lang.Error: Unresolved compilation problem: 
	The declared package "main.java.factorial" does not match the expected package "factorial"

	at factorial.Factorial.<init>(Factorial.java:1)
	at app.ExamplesOfAssertionsTest.lambda$0(ExamplesOfAssertionsTest.java:35)
	at org.junit.jupiter.api.AssertThrows.assertThrows(AssertThrows.java:53)
	... 6 more

